case-studies/DLP.md

```diff
@@ -0,0 +1,60 @@
+* Create a prototype (figma)
+* Usability testing
+* Usability testing synthesis
+* Usability testing insights
+
+---
+
+## Reflections
+
+* Initially, workshops were planned to be a full day in length as would be normal for a face-to-face sprint. After a test session, a full day of remote sprint was discovered to be too long, and the programme was restructured to take place over half day workshops. While the length of sprint exercises is malleable to the length of time available, we found that many could be cut significantly while still being productive.
+* The three problem areas we began with focused on youth services issues, but were otherwise very broad. This created a wide range of areas that could be targeted for a digital solution and it became difficult to facilitate a decision with such a range of voices in the room. The GV design sprint recommends appointing a "Decider" who has final say. To keep things democratic this role was avoided for MBL, but in hindsight it may have been useful and is recommended if there is someone on the sprint team who would be appropriate for the role. In lieu of a Decider, dot voting is a good way to make choices. It may be wise with a smaller sprint team to allot more dot votes, otherwise it can be hard to identify a favourite amongst several solution options.
+* Storyboarding is the last exercise in the sprint as suggested by GV and involves drawing frames of the solution step-by-step as the user would see them. As this is a group activity and time consuming it was too difficult to replicate effectively during MBL using remote, online tools. Instead we used a 'user flow' exercise where the equivalent frame is described on a note in text. The frames could show different user journeys through the solution via arrows from note to note.
+* GV recommends creating "how might we" questions during 'Ask the Experts'. However without an expert or with a broader area to ask around, we found it is fine to ask questions loosely around the problem area itself, or if you have done a mapping exercise, ask questions about stages on the map.
+* The 'Notes' part of the sketching exercises can be less useful if you're running a quicker sprint - people usually have the previous exercises still at the forefront of their minds.
+* 'Choosing a Target' can easily become a lengthy discussion if allowed, as can other points where significant choices are to be made. If you're short on time try to avoid the discussion and head straight for dot voting. Useful points or avenues for exploration will have to be left behind, especially with a broad problem area. Make note of them for another time.
+* If you are reflecting as a group on an exercise such as 'How Might We', reading out every note is time consuming and can be boring when the participants could easily read themselves. Instead it may be worth giving a small amount of time to read through a lot of info and reflect personally, then continue discussion from there.
```

Comment:
Perhaps this one could be give a more positive spin - e.g. "Focusing on the riskiest, most relevant, or insightful HMW's can speed up the flow if there is limited time..."